apps/site/src/_pages/docs/1_blocks/02_develop.mdx

```diff
@@ -403,7 +403,10 @@ Once you’ve finished writing your block, run `yarn build` or `npm run build`.
 
 This will produce a compiled version of your code in the `dist` folder, along with a metadata file describing your block (`block-metadata.json`).
 
-It is worth updating the `blockprotocol` object in `package.json` to include your own `icon`, `image`, and `examples` for your block. These will automatically be included in the `block-metadata.json` produced after running `yarn build` or `npm run build`.
+It is worth updating the `blockprotocol` object in `package.json` to include your own `icon`, and `image` for your block.
+These will automatically be included in the `block-metadata.json` produced after running `yarn build` or `npm run build`.
+
+In addition to the above, updating the `example-graph` (either `./src/example-graph.ts` or `./example-graph.json` depending on approach) is a great way to help users understand how to use your block, and how it functions. Many environments (such as the [Þ Hub]) will insert this data when users are viewing blocks.
```

Comment:
```suggestion
In addition to the above, updating the `example-graph` (either `./src/example-graph.ts` or `./example-graph.json` depending on approach) is a great way to help users understand how to use your block, and how it functions. Many environments (such as the [Þ Hub]("https://blockprotocol.org/hub")) will insert this data when users are viewing blocks.
```

Reply:
For some reason I thought this was injected and globally defined..

I could have sworn I saw it in some of the other `mdx` files but after searching I can't see that. Will fix, thanks!